playbook/workshops/contrib/ajar/README.md

```diff
@@ -234,4 +234,179 @@ You've finished Part 1!
 
 ![](img/celebrate.gif)
 
-# More coming soon...
+
+## Part 2
+
+Welcome to part 2 of the Ajar.io workshop!
+
+To recap, in [part 1](#part-1) you learned about:
+
+- **JavaScript variables**: used to store data values. Declared using the `var`
+  keyword. You can make variables like this:
+
+```js
+var x = 10 * 2;
+var name = "Bogdan";
+```
+
+- **Drawing on the canvas**: the HTML `<canvas>` element is used to draw
+  graphics, on the fly, via scripting (usually JavaScript). You obtain the 
+  drawing context of the `<canvas>` element by calling the _function_ (more on
+  what that means today) `getContext("2d")` of the `<canvas>` element. You use
+  the drawing context to draw shapes on the canvas.
+
+In this part you will learn about **functions**.
+
+### What are functions?
+
+In part 1, you wrote all your code outside of any functions. This means that
+your code runs from *top to bottom*. This was okay for a simple task such as
+displaying one circle on screen. For more complex scenarios we will need 
+**functions**.
+
+**A JavaScript function is a block of code designed to perform a particular
+task.**
+
+A function looks like this:
+
+```js
+function name() {
+    // code to be executed goes here <-- that is a comment
```

Comment:
What if you add some code to run as an example? (ex. `return "Brian Fox";`)

We demonstrate calling a function on L288, but nothing will happen if they are following along in their terminal.